miniapps/tools/load-dc.cpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+//
+//    -------------------------------------------------------------------
+//    Load DC Miniapp:  Visualize fields saved via DataCollection classes
+//    -------------------------------------------------------------------
+//
+// This miniapp loads and visualizes (in GLVis) previously saved data using
+// DataCollection sub-classes, see e.g. Example 5/5p. Currently, only the
+// VisItDataCollection class is supported.
+//
+// Compile with: make load-dc
+//
+// Serial sample runs:
+//   > load-dc -r ../../examples/Example5
+//
+// Paralle sample runs:
+//   > mpirun -np 4 load-dc -r ../../examples/Example5-Parallel
+
+#include "mfem.hpp"
+
+using namespace std;
+using namespace mfem;
+
+int main(int argc, char *argv[])
+{
+#ifdef MFEM_USE_MPI
+   MPI_Session mpi;
+   if (!mpi.Root()) { mfem::out.Disable(); mfem::err.Disable(); }
+#endif
+
+   // Parse command-line options.
+   const char *coll_name = NULL;
+   int cycle = 0;
```

Comment:
Should the default for cycle be ``-1``.  

The comment for ``cycle`` in ``DataCollection`` is that data collections are treated as time-dependent when ``cycle >= 0``.

https://github.com/mfem/mfem/pull/395/files#diff-cfa632e40ac73ff17e30b42d798084bcR69

Reply:
Right now the miniapp supports only `VisItDataCollection` which always uses `cycle >= 0` with `cycle = 0` as the default.